homeassistant/components/media_player/epson.py

```diff
@@ -0,0 +1,320 @@
+"""
+Support for Epson projector
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/
+
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+import async_timeout
+import asyncio
+import aiohttp
+
+from homeassistant.components.media_player import (
+    PLATFORM_SCHEMA, MediaPlayerDevice, SUPPORT_TURN_ON, SUPPORT_TURN_OFF,SUPPORT_SELECT_SOURCE, MEDIA_PLAYER_SCHEMA, DOMAIN,
+    SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_STEP, SUPPORT_NEXT_TRACK, SUPPORT_PREVIOUS_TRACK
+)
+
+""" ,SUPPORT_PLAY, SUPPORT_PAUSE, SUPPORT_STOP"""
+
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+from homeassistant.const import (ATTR_ENTITY_ID, CONF_HOST,CONF_NAME,CONF_PORT,CONF_SSL, STATE_ON, STATE_OFF, STATE_UNKNOWN, STATE_PLAYING)
+
+import homeassistant.util as util
```

Comment:
Keep the import [ordered](https://www.python.org/dev/peps/pep-0008/#imports). `isort` can help.